api/config/__init__.py

```diff
@@ -23,6 +25,10 @@ def __init__(self, **data: Any) -> None:
     def db_connection_string(self):
         return f"{self.db_protocol}://{self.db_username}:{self.db_password}@{self.db_host}/?retryWrites=true&w=majority"
 
+    @property
+    def local_development(self):
+        return self.outside_host == "localhost"
```

Comment:
We will be running the triggers and scheduler in a different way in this project.  I'd like to be able to independently manage triggers, schedules and the api.  This is how it will behave in the cloud deployment as well.

Reply:
I am working on this branch to separate start/stop of triggers with the api using the Makefile.